rails/docker/dev/run.sh

```diff
@@ -39,7 +39,12 @@ if [ ! -f $DB_CONFIG ]; then
 fi
 
 if [ "$RAILS_ENV" = "production" ]; then
+  # Build react components, prod version
+  cd react-components && npm install && npm run build && cd ..
```

Comment:
Same question here and on line 47 - shouldn't this use `npm ci` instead of `npm install` so that it uses the lockfile?